frappe/www/list.py

```diff
@@ -169,6 +169,11 @@ def get_list(doctype, txt, filters, limit_start, limit_page_length=20, ignore_pe
 			for f in meta.get_search_fields():
 				if f == 'name' or meta.get_field(f).fieldtype in ('Data', 'Text', 'Small Text', 'Text Editor'):
 					or_filters.append([doctype, f, "like", "%" + txt + "%"])
+				elif f == 'items':
```

Comment:
Hi @rmehta ! Any suggestion on this or something similar in frappe to look into? Thanks!